packages/cipherstash-proxy/src/eql/mod.rs

```diff
@@ -106,3 +108,28 @@ where
     let s = ident.to_string();
     serializer.serialize_str(&s)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{Identifier, Plaintext};
+
+    #[test]
+    pub fn plaintext_json() {
+        let identifier = Identifier::new("table", "column");
+        let pt = Plaintext {
+            identifier,
+            plaintext: "plaintext".to_string(),
+            version: 1,
+            for_query: None,
+        };
+
+        let value = serde_json::to_value(&pt).unwrap();
```

Comment:
Do we serialize the `Plaintext` struct outside of tests? Might be more useful to test against the encrypted payload since that's what gets stored in PG.

Reply:
Good call